controllers/bootstrap/init.go

```diff
@@ -1374,11 +1376,48 @@ func (b *Bootstrap) updateApprovalMode() error {
 		return err
 	}
 
-	if err = b.UpdateCsOpApproval(); err != nil {
+	if err = b.UpdateOpApproval("ibm-common-service-operator"); err != nil {
 		klog.Errorf("Failed to update common service operator subscription: %v", err)
 		return err
 	}
 
+	if err = b.UpdateOpApproval("operand-deployment-lifecycle-manager-app"); err != nil {
+		klog.Errorf("Failed to update operand-deployment-lifecycle-manager-app subscription: %v", err)
+		return err
+	}
+
+	if err = b.UpdateOpApproval("ibm-namespace-scope-operator"); err != nil {
+		klog.Errorf("Failed to update namespace-scope-operator subscription: %v", err)
+		return err
+	}
+
+	if err = b.UpdateOpApproval(constant.ICPOperator); err != nil {
+		if !errors.IsNotFound(err) {
+			klog.Errorf("Failed to update %s subscription: %v", constant.ICPOperator, err)
+			return err
+		}
+		klog.Infof("%s not installed, skipping", constant.ICPOperator)
+
+	}
+
+	if err = b.UpdateOpApproval(constant.ICPPICOperator); err != nil {
+		if !errors.IsNotFound(err) {
+			klog.Errorf("Failed to update %s subscription: %v", constant.ICPPICOperator, err)
+			return err
+		}
+		klog.Infof("%s not installed, skipping", constant.ICPPICOperator)
+
+	}
+
+	if err = b.UpdateOpApproval(constant.ICPPKOperator); err != nil {
+		if !errors.IsNotFound(err) {
+			klog.Errorf("Failed to update %s subscription: %v", constant.ICPPKOperator, err)
+			return err
+		}
+		klog.Infof("%s not installed, skipping", constant.ICPPKOperator)
```

Comment:
Hi @qpdpQ 

Could you try to use lower verbose level for the information log?
I think it would be better to reduce some logs for debug convenience.

Here is an example of lower verbose level
https://github.com/IBM/ibm-common-service-operator/blob/master/controllers/bootstrap/init.go#L1394
```
klog.V(2).Infof("waiting for resource ready with kind: %s, apiGroupVersion: %s", kind, apiGroupVersion)
```